### ISSUE TITLE
$10k project idea #3

### ISSUE THREAD

**Issue by mgifford** [canada-ca/devex#82]
WCAG 2.1 is being adopted in Europe & Australia. What are the implications for the Government of Canada and the existing Web Experience Toolkit?

Evaluate what's new in the guidelines http://davidmacd.com/blog/wcag-2.1-quick-guide.html and where are the areas for improvement that could be adopted. 

Contribute those back as issues https://github.com/wet-boew/wet-boew/issues

**Comment by MaryBethBaker** [comment]
From Medium post: https://medium.com/@mgifford/i-would-definitely-update-it-with-or-without-covid-19-e326e1882603

Update on issue from May 2020

#3 —WET & WCAG 2.1 AA, this all still needs to be done. There has been a little progress, but WCAG 2.2 should be finalized this fall.

**Comment by mgifford** [comment]
I think this is the meta issue for WET-BOEW:
https://github.com/wet-boew/wet-boew/issues/8602

The first 2.2 draft is up here https://www.w3.org/TR/WCAG22/
